Add Description on H2 Console Removal to what-has-changed page (#2873)
wso2/docs-is: en/docs/setup/migrating-what-has-changed.md

proxy_context_path="abc"
```

> ## Removed H2 Console

Reviewer: ```suggestion
## Removed H2 console
```

> The H2 console packed with the embedded H2 database bears the risk of having security vulnerabilities due to remote code execution. Since there is no valid usage of the H2 console feature in the production environments and is only useful during development tasks, it has been removed from the WSO2 IS 5.12.0. As an alternative, use standard database client software for development or testing purposes.  

Reviewer: ```suggestion
The H2 console, which is packed with the embedded H2 database can cause security vulnerabilities due to remote code execution. Since there is no valid usage of the H2 console in production environments and is only useful during development tasks, it has been removed from WSO2 IS 5.12.0. As an alternative, you can use standard database client software for development or testing purposes.  
```
